api/actions/actionsRouter.js

```diff
@@ -153,16 +154,11 @@ router.get('/', function (req, res) {
  *                  strike: true
  *                  created_at: "2022-01-26 15:33:34.945832-07"
  *                  updated_at: "2022-01-26 15:33:34.945832-07"
+ *      '400':
+ *        description: Bad Request, Action with that Id does not exist
  */
-router.get('/:id', function (req, res) {
-  const id = req.params.id;
-  Actions.findById(id)
-    .then((action) => {
-      res.status(200).json(action);
-    })
-    .catch((err) => {
-      res.status(500).json({ message: err.message });
-    });
+router.get('/:id', checkActionTicketExists, (req, res) => {
```

Comment:
Looks like middleware was added to optimize the code and keep it dry; the bulk of the work is done within the middleware. Nice!